extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -219,6 +221,25 @@ export class SafeframeHostApi {
     return attributes;
   }
 
+  /**
+   * Returns the canonical URL of the page, if the publisher allows
+   * it to be passed.
+   * @return {string|boolean}
+   */
+  maybeGetCanonicalUrl() {
+    // Don't allow for referrer policy same-origin,
+    // as Safeframe will always be a different origin.
+    // Don't allow for no-referrer.
+    if (!!this.win_.document.querySelector(
```

Comment:
Let's save two query selectors either by changing this to:  querySelector("meta[name='referrer'][content='same-origin'], meta[name='referrer'][content='no-referrer']") 
OR
const metaReferrer = this.win_.document.querySelector("meta[name='referrer']");
if (metaReferrer && (metaReferrer.getAttribute('content') == 'same-origin' || metaReferrer.getAttribute('content') == 'no-referrer')) return false;

Reply:
done